examples/duet/dcgan/DCGAN_Syft_Data_Scientist.ipynb

```diff
@@ -7,9 +7,7 @@
    "source": [
```

Comment:
<p>Same</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Thanks for the comment!